test/e2e/kubectl/kubectl.go

```diff
@@ -2069,6 +2073,97 @@ metadata:
 			gomega.Expect(outBuiltIn).To(gomega.Equal(outStatusSubresource))
 		})
 	})
+
+	ginkgo.Describe("Kubectl rollout", func() {
+		deploymentYaml := commonutils.SubstituteImageName(string(readTestFileOrDie(labeledDeploymentFilename)))
+		ginkgo.AfterEach(func() {
+			cleanupKubectlInputs(deploymentYaml, ns, "labels_foo=labels_bar")
+		})
+		ginkgo.It("undo should rollback and update deployment env", func(ctx context.Context) {
+			var err error
+			// create deployment
+			_ = e2ekubectl.RunKubectlOrDieInput(ns, deploymentYaml, "apply", "-f", "-")
+
+			replica := int32(1)
+			d := &appsv1.Deployment{
```

Comment:
Why not just use the already present httpdDeployment1Filename instead?

Reply:
It's because we want ot test some labels/anntations will be rolled back correctly https://github.com/kubernetes/kubernetes/pull/119915#discussion_r1300388454. And I'm not sure if we can just add them to `httpdDeployment1Filename`